rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -124,7 +124,11 @@
      * WARNING: NarayanaLRAClient is an internal utility class and is subject to change
      * (but it will be made available with improved guarantees in a subsequent release).
      */
-    private static final long CLIENT_TIMEOUT = Long.getLong("lra.internal.client.timeout", 10);
```

Comment:
Would not be good to have one system property to define all client timeout values at once and on top of that to provide ability to granularly change the particular value?

Reply:
Well that would be more code than I wanted to provide at this stage and we'd have document clearly which properties take precedence over others. I will rename the properties so that we can do what you request in a future revision, something like:

```java
    private static final long CLIENT_TIMEOUT = Long.getLong("lra.internal.client.timeout", 10);
    private static final long START_TIMEOUT = Long.getLong("lra.internal.client.timeouts.tart", 10);
    private static final long JOIN_TIMEOUT = Long.getLong("lra.internal.client.timeout.join", 10);
etc
```

Reply:
ok

Reply:
Actually the change was rather trivial so I incorporated it into the PR.